packages/indexing/src/tables-manager.ts

```diff
@@ -162,6 +154,35 @@ export class TablesManager {
     }
   }
 
+  /**
+   * List and migrate all MID tables
+   */
+  async migrateMidTables(): Promise<void> {
+    const tableNames = await this.listMidTables()
+    await Promise.all(
+      tableNames.map(async (table) => {
+        return await this._migrateMidTable(table)
+      })
+    )
+  }
+
+  /**
+   * Migrate a MID table
+   * @param tableName
+   */
+  async _migrateMidTable(tableName: string): Promise<void> {
+    const columns = await this.dataSource.table(tableName).columnInfo()
+    const relationColumns = Object.keys(columns).filter((name) => {
+      return name.startsWith(CUSTOM_COLUMN_PREFIX)
```

Comment:
I see where we check for this, but not anything that indicates that a column in the table should not start with this prefix, otherwise it will get removed.

Reply:
Only relation fields use this prefix, we don't have any other column that uses it.